### PR TITLE
Header and headeline  for mobile version

### PR DESCRIPTION
**Pull request by souad988**
created header with welcome and menu icon 
created headline section with headline and description 
added socialmedia icons

### CODE REVIEW

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @souad988 , 

### You did a really good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

To highlight:
  - ✔ Meaningful commit messages.
  - ✔ All content is aligned with flexbox.


## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### None.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                        </div>
+                
+                <div class="social_media">
+                    <ul class="links">
+                        <li> <a><img src="../media/icons/github.png" alt="github"/></a></li> 
+                        <li><a><img src="../media/icons/twitter.png" alt="twiter"/></a></li>
+                        <li><a><img src="../media/icons/Linkedin.png" alt="linkdin"/></a></li>
+                        <li><a><img src="../media/icons/angellist.png" alt="angellist"/></a></li>
+                    </ul>    
+                </div>
```
Comment (Whoistolu) [inline src/index.html:50]: - [ ] [REQUIRED] It is nice to see that you have embedded the icons in an unordered and ordered list, however, the link to the icons are broken as they do not display on your page. Please refer to the design images above for proper clarity,

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @souad988 ,

Good job so far!
Thank you for implementing some of the recommended changes by the last reviewer however, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] [OPTIONAL] Kindly ensure that the first letter of your commit message starts with a capital letter. This is will groom your into writing professional commit messages. See bellow:
### Your commit messages
your-commit-message

### Standard Commit Message Template
proper-commit-messages



_Please check the inline comments under the review for sections to improve._

Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                    </div>
+                </div>
+                <div class="headline-description">
+                    <p>
+                        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        don’t hestiate to contact me.
+                    </p>
+                </div>
+           </div>
+           <div class="social_media">
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply src/index.html]: - [ ] [REQUIRED] It is nice to see that you have implemented the unordered list and the ordered list however, the social media icons do not display. Please refer to this [comments](https://github.com/souad988/Portfolio/pull/2#pullrequestreview-846348945) for more clarity.

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+           
+           <header class="header">
+            <div class="header welcome">welcome</div>
+            <nav class="header menu_icon"><img src="../media/Icons/Union.png" alt="menu icon"/></nav>
+           </header>
+           
+           <section class="home">
+                
+                        <div class="headlines">
+                            <h1 class="headline-1">Hey there. I’m Souad</h1>
```
Comment (FrederickMih) [inline src/index.html:25]: - Hi @souad988, great job for making these changes

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                                don’t hestiate to contact me.
+                            </p>
+                        </div>
+                
+                        <div class="social_media">
+                            <ul class="links">
+                                <li><a href="https://github.com/souad988"><img src="../media/icons/github.png" alt="github"/></a></li> 
+                                <li><a href="https://twitter.com/souadElmansouri"><img src="../media/icons/twitter.png" alt="twiter"/></a></li>
+                                <li><a href="www.linkedin.com/in/souad-el-mansouri-4725491a0"><img src="../media/icons/Linkedin.png" alt="linkdin"/></a></li>
+                                <li><a href="https://www.angellist.com/"><img src="../media/icons/angellist.png" alt="angellist"/></a></li>
```
Comment (FrederickMih) [inline src/index.html:41]: - Great job for fixing this issue addressed by the previous reviewers.

**Review by FrederickMih — APPROVED** [review]
# Hi @souad988 , 

#### NB.  As a reminder, please add a descriptive README file for this project. Your README file is the first impression a visitor takes about your project. It will be nice to make it more descriptive and professional. You should use [this template](https://github.com/microverseinc/readme-template) and later customize it for your project.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**